src/__internal__/validation-message/validation-message.component.tsx

```diff
@@ -10,12 +10,15 @@ export interface ValidationMessageProps {
   /** Indicate that warning has occurred
   Pass string to display hint with warning */
   warning?: boolean | string;
+  /** Used to set font size if large size of a component is used.  */
+  sizeIsLarge?: boolean;
```

Comment:
**question:** (non-blocking):After checking the DS and the corresponding figma, I can see there is a need for a larger text variant, which I personally don't see the need for especially with such a minor `px` difference. But I agree, we should stick to the figma and add what's there. Instead of a`boolean`, could we perhaps use a prop like `fontSize` with a type of `"medium"` | `"large"`, with `"medium"` being the default?

Reply:
Decided that we're doing this in a separate piece of work so I'll resolve this.